pkg/apis/events/v1alpha1/cloudschedulersource_validation_test.go

```diff
@@ -203,7 +203,7 @@ func TestCloudSchedulerSourceSpecValidationFields(t *testing.T) {
 			},
 		},
 		want: func() *apis.FieldError {
-			fe := apis.ErrMissingField("secret.name")
+			fe := apis.ErrMissingField("name", "secret.name")
```

Comment:
hmm, why this changed? Seems like just "secret.name" should be there... same as below.